composer.json

```diff
@@ -23,9 +23,9 @@
     }
   ],
   "require": {
-    "php": ">=5.5.0",
-    "doctrine/orm": "2.5.*|2.6.*|2.7.*",
-    "doctrine/inflector": "^1.1"
+    "php": "^7.2",
+    "doctrine/orm": "2.7.*",
```

Comment:
Dropping support for doctrine/orm 2.6 will make this unavailable for L5.8 users. But I guess thats probably fine? They can proably just use 1.2 instead. 

L5.8 is no longer maintained either so..

For reference, 1.4 which is last to support L5:
https://github.com/laravel-doctrine/orm/blob/1.4/composer.json

Reply:
> But I guess thats probably fine?

yes, that's what the idea was

Reply:
Is there any chance of an release soon?

The current version also conflicts with doctrine/common 3. I can create a PR for that when there is a chance fo a new release (1.2)?

Reply:
I will take a look at this now.

Reply:
I pushed branch 1.x with some these changes. 
Tried to add support for doctrine/orm 2.8 and PHP 8 but was stopped by some issues.

Have no time to fix those issues now, but I am open for a PR @bernhard-efler :) If you would like to try then update .github/workflows/ci.yml, should probably add doctirne/orm 2.8 here?

Please open a new issue if you would like to discuss this further.